.github/workflows/archlinux-test.yml

```diff
@@ -0,0 +1,11 @@
+name: Arch Linux Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run Arch Linux Test
+      run: sudo make -C scripts/ci archlinux
```

Comment:
You might need to update [scripts/ci/Makefile](https://github.com/checkpoint-restore/criu/blob/47611e668e49fe0b17c39d9b2b2d83b04b2b4cb6/scripts/ci/Makefile) as well.

Reply:
Thanks I missed that. My bad :)